modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -0,0 +1,5471 @@
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `10000` (10s)
+
+---
+
+=== admin_api_require_auth
+
+Whether Admin API clients must provide HTTP basic authentication headers.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== aggregate_metrics
+
+Enable aggregation of metrics returned by the xref:reference:internal-metrics-reference.adoc[`/metrics`] endpoint. Aggregation can simplify monitoring by providing summarized data instead of raw, per-instance metrics. Metric aggregation is performed by summing the values of samples by labels and is done when it makes sense by the shard and/or partition labels.
+
+*Type*: boolean.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `None`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== alter_topic_cfg_timeout_ms
+
+The duration, in milliseconds, that Redpanda waits for the replication of entries in the controller log when executing a request to alter topic configurations. This timeout ensures that configuration changes are replicated across the cluster before the alteration request is considered complete.
+
+*Unit:* milliseconds
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `5000` (5s)
+
+---
+
+=== append_chunk_size
+
+Size of direct write operations to disk in bytes. A larger chunk size can improve performance for write-heavy workloads, but increase latency for these writes as more data is collected before each write operation. A smaller chunk size can decrease write latency, but potentially increase the number of disk I/O operations.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `16384`
+
+---
+
+=== audit_client_max_buffer_size
+
+Defines the number of bytes allocated by the internal audit client for audit messages. When changing this, you must disable audit logging and then re-enable it for the change to take effect. Consider increasing this if your system generates a very large number of audit records in a short amount of time.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Default:* `16777216`
+
+---
+
+=== audit_enabled
+
+Enables or disables audit logging. When you set this to true, Redpanda checks for an existing topic named `_redpanda.audit_log`. If none is found, Redpanda automatically creates one for you.
+
+*Requires restart:* No
+
+*Optional:* Yes
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== audit_enabled_event_types
+
+List of strings in JSON style identifying the event types to include in the audit log. This may include any of the following - `management, produce, consume, describe, heartbeat, authenticate, schema_registry, admin`.
```

Comment:
```suggestion
List of strings in JSON style identifying the event types to include in the audit log. This may include any of the following: `management, produce, consume, describe, heartbeat, authenticate, schema_registry, admin`.
```